events/race.md

```diff
@@ -56,11 +56,35 @@ Indicates that racers should start racing.
 
 Indicates the racer has passed a gate, possibly completed a lap, 
 
-- TODO: Rename this or break it into multiple events? `racer_completed_lap` and `racer_passed_gate`
-- TODO: This doesn't indicate the gate, should it?
+### Properties
+- `fast` [boolean]: Racer's fastest lap
+- `streak` [boolean]: Indicates if the racer is on a streak
+- `valid` [boolean]: Valid gate pass
+- `transponder` [string]: The transponder ID of the racer
+- `gate` [string]: The id of the gate the racer passed
+- `gate_type` [string start|checkpoint|finish]: The type of gate the racer passed
 
 ```json
-{"cmd":"event","evt":"race","type":"racer_passed_gate","fast":false,"streak_laps":0,"lap":true,"transponder":"123",[...protocol properties]}
+{"cmd":"event","evt":"race","type":"racer_passed_gate","fast":false,"streak":true,"lap":true,"transponder":"123","gate_id":"1","gate_type":"start",[...protocol properties]}
+```
+
+
+## Race Standings
```

Comment:
What about a post race standing? Which gives positions etc.
I grant that this could be too much data though :/

Reply:
What is etc? Are you just wanting race standings data + positions?

Reply:
I'd use position, name and lap total. Anything else is nice but not what I'd use :)